docs/react-jss.md

```diff
@@ -6,8 +6,8 @@ Benefits compared to the lower level core:
 
 - Dynamic Theming - allows context based theme propagation and runtime updates.
 - Critical CSS extraction - only CSS from rendered components gets extracted.
-- Lazy evaluation - Style Sheets are only created when a component gets mounted.
-- Static part of a Style Sheet is shared between all elements.
+- Lazy evaluation - Style Sheets are created when a component mounts and removed when it's unmounted.
```

Comment:
I was rewriting those sentences just recently to get rid of all the grammarly errors/warnings. I am not sure whats the right version but it says this


<img width="1204" alt="screen shot 2019-01-13 at 23 07 58" src="https://user-images.githubusercontent.com/52824/51091225-33967400-1788-11e9-9779-09ef0e4c14e8.png">

Reply:
Also there are the same points in features.md

Reply:
Changing the features.md shouldn't be in this PR